modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/build.gradle

```diff
@@ -31,10 +31,15 @@ dependencies {
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "4.4.0"
 #end
 	compileOnly group: "com.liferay.portal", name: "com.liferay.util.bridges", version: "2.0.0"
```

Comment:
hey @lawrence-lee  I think we need to do the same thing for `com.liferay.util.bridges` update the templates with the 7.0.x, 7.1.x and 7.2.x versions.  in the future, this is going to go away with improved TP plugin.

Reply:
Closing PR. will resend with versions from the BOMs